### DISCUSSION TITLE
Disable Alt-Key feature

### DISCUSSION THREAD

**Discussion by Dinal2019** [sumatrapdfreader/sumatrapdf#4165] [Q&A]
How can i disable Alt key feature

**Comment by kjk** [comment]
Assuming you're referring to `Alt` key triggering menu: you can't. This is a standard Windows behavior.

**Comment by Dinal2019** [comment]
Thank for your reply. Then how to disable open folder in explorer from the menu. It there any settings for it.

**Reply by kjk** [reply]
You can configure to run in restricted mode https://www.sumatrapdfreader.org/docs/Configure-for-restricted-use and set `DiskAccess = 0`

This will disable all functionality that requires disk access.

There is no way to disable just that one menu item.

**Comment by Dinal2019** [comment]
Hi, Good Morning. Many thanks for your help. It works, this was i am looking for. Appreciated

**Comment by Dinal2019** [comment]
Is there any way to hide properties?

**Comment by kjk** [comment]
No, there's no way to hide properties.

**Comment by Dinal2019** [comment]
Many thanks for you supportPlease consider in your next updateAtleast the filenameSent from my iPhoneOn 3 Apr 2024, at 12:52 PM, Krzysztof Kowalczyk ***@***.***> wrote:﻿
No, there's no way to hide properties.

**Reply by GitHubRulesOK** [reply]
Hmmm there are several ways a user can see the current file path and name (and access that folder) even when in fully locked down ini state (all set to 0), so pointless removing the name from dialog. you would need to go deeper to block all user friendly functionality.

So If you need a secured PDF application for "kiosk" client usage you may find MS edge is the windows designed default kiosk app. Several commercial PDF readers are specifically targeted at corporate users.
